File: 01.CSharp-Programming-Basics/04.Simple-Operations/HomeWork/SimpleOperations/YardGreening/YardGreening.cs
using System;

namespace YardGreening
{
    class YardGreening
    {
        static void Main(string[] args)
        {
            double squareMeters = double.Parse(Console.ReadLine());

            double allSum = squareMeters * 7.61;
            double discount = 0.18 * allSum;

            double finalSum = allSum - discount;

            Console.WriteLine($"The final price is: {finalSum:f2} lv.");
            Console.WriteLine($"The discount is: {discount:f2} lv.");
        }
    }
}

File: 02.JS-Programming-Basics/01.First-Steps-In-Coding/Lab/YardGreening.js
function yardGreening(inPut){

    let meters = Number(inPut);

    let totalPrice = meters * 7.61;
    let discount = totalPrice * 0.18;
    totalPrice -= discount;

    console.log(`The final price is: ${totalPrice} lv.`)
    console.log(`The discount is: ${discount} lv.`)
}

yardGreening("550");
yardGreening("150");